lockfile/verification/src/satisfiesPackageManifest.ts

```diff
@@ -36,10 +37,11 @@ export function satisfiesPackageManifest (
     existingDeps = pickNonLinkedDeps(existingDeps)
     specs = pickNonLinkedDeps(specs)
   }
-  if (!equals(existingDeps, specs)) {
+  const specsDiff = diffFlatRecords(specs, existingDeps)
+  if (!isEqual(specsDiff)) {
```

Comment:
Although `ramda` already has `difference` and `equals`, they are more verbose than necessary for flat objects.